packages/cxl-ui/package.json

```diff
@@ -10,6 +10,7 @@
     "directory": "packages/cxl-ui"
   },
   "dependencies": {
+    "@appnest/masonry-layout": "^2.0.8",
```

Comment:
WP ships Masonry JS in core https://github.com/WordPress/WordPress/blob/master/wp-includes/js/masonry.min.js

I think we should probably keep aligned with core. Can you attempt adding this as a new dependency in Storybook?

Reply:
Regardless, @appnest/masonry-layout is also a very nice find.

> Lightweight: Super small (1kb minified & gzipped)

Whereas WP core Masonry script is 23 kB minified, and 7.69 kB in transit.

This might actually shift the scales back to existing web component's favor :thinking:

Reply:
Hi Leho, thanks. 

I'll leave room for a moment of thought here, too, while preparing the sidebar draft and then check back. 

I liked this webcomponent because of how nice it handles the slotted items. 

Although incorporating the best-known masonry library would require some wrapping to select those elements, if it turns out to be the better way wordpress-wise, i'll give it a try. I've already seen another webcomponent that does just that so i'll try it first or get inspiration from there.